api/fittings/lisk_cache.js

```diff
@@ -13,9 +13,26 @@
  */
 'use strict';
 
-var modules = require('../../helpers/swagger_module_registry');
 var debug = require('debug')('swagger:lisk:cache');
+var modules = require('../../helpers/swagger_module_registry');
 
+/**
+ * Description of the module.
+ * @module
+ * @requires debug
+ * @requires helpers/swagger_module_registry.getCache
+ * @requires helpers/swagger_module_registry.getLogger
+ * @todo: add description of the module
+ */
+
+/**
+ * Description of the function.
+ * @func create
+ * @param {Object} fittingDef - Description of the param.
+ * @param {Object} bagpipes - Description of the param.
+ * @returns {function} lisk_cache
```

Comment:
Is there a way to provide a link from this to the function below in the generated documentation?

Reply:
I could not find a way to link it directly, but one could add an additional `@see`-tag, that will add a link to the function in the documentation

Reply:
not if we are using namespaces, it seems (normally inlune links are done with `@link`)